lib/urldata.h

```diff
@@ -1822,6 +1822,7 @@ struct UserDefined {
 #ifdef USE_ECH
   int tls_ech;      /* TLS ECH configuration  */
 #endif
+  long upload_flags;
```

Comment:
We try to keep the fields order roughly by size, so put this further up in the struct. Since we only use five bits here, maybe we can stick to using an `unsigned char` for this data?